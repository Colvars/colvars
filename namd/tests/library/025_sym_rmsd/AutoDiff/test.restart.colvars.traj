# step         rmsd                   rmsd_sym               rmsd_min              
          20    1.67010101996501e+00   1.06269628600354e+00   1.06269628600354e+00 
          21    2.04433892429578e+00   1.60292722878381e+00   1.60292722878381e+00 
          22    2.06240612835228e+00   1.64334625236222e+00   1.64334625236222e+00 
          23    1.72479860434134e+00   1.21752251761603e+00   1.21752251761603e+00 
          24    1.45281664722968e+00   8.31805423263468e-01   8.31805423263468e-01 
          25    1.62193082908377e+00   1.13819710156539e+00   1.13819710156539e+00 
          26    1.91428771037449e+00   1.54849490838439e+00   1.54849490838439e+00 
          27    1.93241576009644e+00   1.57970290898642e+00   1.57970290898642e+00 
          28    1.64032456570087e+00   1.19887684608299e+00   1.19887684608299e+00 
          29    1.42831104802460e+00   8.67878207872020e-01   8.67878207872020e-01 
          30    1.71179394524434e+00   1.27004409838273e+00   1.27004409838273e+00 
          31    2.10204997402056e+00   1.75291645597024e+00   1.75291645597024e+00 
          32    2.16523333046785e+00   1.81717671975520e+00   1.81717671975520e+00 
          33    1.85227206897975e+00   1.41097165384593e+00   1.41097165384593e+00 
          34    1.51283191580260e+00   8.90242598276096e-01   8.90242598276096e-01 
          35    1.65809367768181e+00   1.09671669300779e+00   1.09671669300779e+00 
          36    2.03497663158023e+00   1.59379893021680e+00   1.59379893021680e+00 
          37    2.17858401892649e+00   1.75248645511788e+00   1.75248645511788e+00 
          38    1.99046986313608e+00   1.48057223445693e+00   1.48057223445693e+00 
          39    1.64708037387980e+00   9.18991758873705e-01   9.18991758873705e-01 
          40    1.57261614568718e+00   7.26619298562136e-01   7.26619298562136e-01 
