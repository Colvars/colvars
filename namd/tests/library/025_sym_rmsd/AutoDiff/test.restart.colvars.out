colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # sourceTclFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rmsd"
colvars:   Initializing a new "rmsd" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = on
colvars:       # rotateToReference = on
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       Within atom group "atoms":
colvars:       Initializing atom group "fittingGroup".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "fittingGroup" defined with 9 atoms requested: total mass = 70.071, total charge = -0.09.
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "../Common/da.pdb"
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use fittingGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 4 atoms requested: total mass = 15.035, total charge = 0.
colvars:     The option "fittingGroup" (alternative group for fitting) was enabled: Jacobian derivatives of the RMSD will not be calculated.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da_flipped.pdb"
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 1 [default]
colvars:   # hardLowerBoundary = on [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rmsd_sym"
colvars:   Initializing a new "rmsd" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = on
colvars:       # rotateToReference = on
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       Within atom group "atoms":
colvars:       Initializing atom group "fittingGroup".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "fittingGroup" defined with 9 atoms requested: total mass = 70.071, total charge = -0.09.
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "../Common/da.pdb"
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use fittingGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 4 atoms requested: total mass = 15.035, total charge = 0.
colvars:     The option "fittingGroup" (alternative group for fitting) was enabled: Jacobian derivatives of the RMSD will not be calculated.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da_flipped.pdb"
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:     atomPermutation = { 0, 2, 3, 1 }
colvars:     atomPermutation = { 0, 3, 1, 2 }
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 1 [default]
colvars:   # hardLowerBoundary = on [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rmsd_min"
colvars:   Initializing a new "rmsd" component.
colvars:     # name = "r1"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = on
colvars:       # rotateToReference = on
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       Within atom group "atoms":
colvars:       Initializing atom group "fittingGroup".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "fittingGroup" defined with 9 atoms requested: total mass = 70.071, total charge = -0.09.
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "../Common/da.pdb"
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use fittingGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 4 atoms requested: total mass = 15.035, total charge = 0.
colvars:     The option "fittingGroup" (alternative group for fitting) was enabled: Jacobian derivatives of the RMSD will not be calculated.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da.pdb"
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   Initializing a new "rmsd" component.
colvars:     # name = "r2"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = on
colvars:       # rotateToReference = on
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       Within atom group "atoms":
colvars:       Initializing atom group "fittingGroup".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "fittingGroup" defined with 9 atoms requested: total mass = 70.071, total charge = -0.09.
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "../Common/da.pdb"
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use fittingGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 4 atoms requested: total mass = 15.035, total charge = 0.
colvars:     The option "fittingGroup" (alternative group for fitting) was enabled: Jacobian derivatives of the RMSD will not be calculated.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da_flipped.pdb"
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   Initializing a new "rmsd" component.
colvars:     # name = "r3"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = on
colvars:       # rotateToReference = on
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       Within atom group "atoms":
colvars:       Initializing atom group "fittingGroup".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "fittingGroup" defined with 9 atoms requested: total mass = 70.071, total charge = -0.09.
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "../Common/da.pdb"
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use fittingGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 4 atoms requested: total mass = 15.035, total charge = 0.
colvars:     The option "fittingGroup" (alternative group for fitting) was enabled: Jacobian derivatives of the RMSD will not be calculated.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da_flipped2.pdb"
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   All components initialized.
colvars:   This colvar uses a custom function.
colvars:   Expecting colvar value of type scalar number.
colvars:   # period = 0 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 1 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 3 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rmsd_sym }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 0 }
colvars:   # targetCenters = { 0 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 100
colvars:   # decoupling = off [default]
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "rmsd_sym" will be rescaled to 100 according to the specified width (1).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (13 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - Symmetry-adapted RMSD:
colvars:   Ebrahimi2022 https://doi.org/10.1021/acs.jctc.1c01235
colvars: 
colvars: - Colvars module:
colvars: - Colvars-NAMD interface:
colvars: - Harmonic colvar bias implementation:
colvars: - Moving frame of reference:
colvars: - Optimal rotation via flexible fitting:
colvars: - rmsd colvar component:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - NAMD engine:
colvars:   Phillips2020 https://doi.org/10.1063/5.0014475
colvars: 
colvars: - Custom functions (Lepton):
colvars:   n/a 
colvars: 
colvars: updating target temperature (T = 0 K).
colvars: ----------------------------------------------------------------------
colvars: Loading state from file "test.colvars.state".
colvars:   Restarting collective variable "rmsd" from value: 1.6701
colvars:   Restarting collective variable "rmsd_sym" from value: 1.0627
colvars:   Restarting collective variable "rmsd_min" from value: 1.0627
colvars:   Restarting harmonic bias "harmonic1" from step number 20.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (13 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: updating target temperature (T = 0 K).
colvars: Re-initialized atom group for variable "rmsd":0/0. 4 atoms: total mass = 15.035, total charge = 0.
colvars: Re-initialized atom group for variable "rmsd_sym":0/0. 4 atoms: total mass = 15.035, total charge = 0.
colvars: Re-initialized atom group for variable "rmsd_min":0/0. 4 atoms: total mass = 15.035, total charge = 0.
colvars: Re-initialized atom group for variable "rmsd_min":1/0. 4 atoms: total mass = 15.035, total charge = 0.
colvars: Re-initialized atom group for variable "rmsd_min":2/0. 4 atoms: total mass = 15.035, total charge = 0.
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Saving collective variables state to "test.restart.colvars.state".
